packages/react-native-reanimated/src/hook/useDerivedValue.ts

```diff
@@ -73,11 +73,5 @@ export function useDerivedValue<Value>(
     };
   }, dependencies);
 
-  useEffect(() => {
-    return () => {
-      initRef.current = null;
-    };
-  }, []);
-
```

Comment:
It seems to be ok to remove it since `useSharedValue` doesn't have any cleanup function.